cpp/include/patterns/count_if_e.cuh

```diff
@@ -35,14 +37,14 @@ namespace experimental {
 namespace detail {
 
 // FIXME: block size requires tuning
-int32_t constexpr count_if_e_for_all_low_out_degree_block_size = 128;
+int32_t constexpr count_if_e_for_all_block_size = 128;
```

Comment:
We could leverage the device properties info from the handle and some basic metrics on the sparsity pattern. A good thing to investigate for perf tunning in 0.17.

Reply:
Yeah... I guess thrust may already have some logic to determine this...